nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
I'm a little confused: is the original algorithm, without parallelization, wrong? That is, does it adjust `scale` too often? Or is this a bug introduced by the parallelization?